src/cityreader/cityreader.py

```diff
@@ -16,13 +28,16 @@
 # should not be loaded into a City object.
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    with open('cities.csv', newline="") as csvfile:
+        reader = csv.DictReader(csvfile)
+        for city in list(csv.reader(csvfile))[1:]:
```

Comment:
Very clean, very well done here. I actually put in the string values 'city', 'lat', 'lon'; this is MUCH better!